src/test/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivityTest.java

```diff
@@ -164,4 +171,168 @@ public void getLocationPermissions_withPermissionsAlreadyDenied_returnsFalse() {
 				});
 		}
 	}
+
+	@Test
+	public void onTouchEvent_withVolumeBtnPressed_startsSettingsDialogActivity() {
+		scenarioRule
+			.getScenario()
+			.onActivity(embeddedBrowserActivity -> {
+				Intents.init();
+				//> GIVEN
+				MotionEvent event = mock(MotionEvent.class);
+				when(event.getPointerCount()).thenReturn(2);
+				when(event.getActionMasked()).thenReturn(MotionEvent.ACTION_POINTER_DOWN);
+				when(event.getX(0)).thenReturn((float) 262.99);
+				when(event.getX(1)).thenReturn((float) 280.49);
+
+				embeddedBrowserActivity.onKeyDown(KeyEvent.KEYCODE_VOLUME_DOWN, null);
+				OnTouchListener onTouchListener = embeddedBrowserActivity.onTouchEvent();
+
+				//>THEN
```

Comment:
Might just be misunderstanding how you are thinking about things, but it feels like you might have accidentally switched the `WHEN` and `THEN` labels on these tests...